charts/nri-prometheus/Chart.yaml

```diff
@@ -8,7 +8,7 @@ icon: https://newrelic.com/themes/custom/curio/assets/mediakit/new_relic_logo_ve
 sources:
   - https://github.com/newrelic/nri-prometheus
 
-version: 2.0.0
+version: 1.15.0
```

Comment:
I put this above in the review but I think this is a breaking change because of Helm 2 deprecation... But I am asking in the Slack channel to talk about this as a team.